src/_help/hubs/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
+
+With hubs, your API ecosystem is displayed on a single page, where documentation can be sorted by categories.
+
+Hubs also provide a unified changelog, making it easy to identify all changes across all the documentation it contains at a glance. Search is also global, allowing you to search across all documentation.
```

Comment:
`unified changelog` when the doc on that topic is ready, it will be valuable to add a link here.

Reply:
👍